CRM/Contribute/Form/Contribution/ThankYou.php

```diff
@@ -52,7 +52,7 @@ public function preProcess(): void {
     // Link (button) for users to create their own Personal Campaign page
     if ($linkText = CRM_PCP_BAO_PCP::getPcpBlockStatus($this->getContributionPageID(), 'contribute')) {
       $linkTextUrl = CRM_Utils_System::url('civicrm/contribute/campaign',
-        "action=add&reset=1&pageId=' . $this->getContributionPageID() . '&component=contribute",
+        'action=add&reset=1&pageId=' . $this->getContributionPageID() . '&component=contribute',
```

Comment:
yep, or could also do
```suggestion
        "action=add&reset=1&pageId={$this->getContributionPageID()}&component=contribute",
```